I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 212, in script call
    call _load_reload_game from _call__load_reload_game_1
  File "renpy/common/00keymap.rpy", line 529, in script
    python hide:
  File "renpy/common/00start.rpy", line 212, in script call
    call _load_reload_game from _call__load_reload_game_1
  File "renpy/common/00keymap.rpy", line 529, in script
    python hide:
  File "renpy/common/00keymap.rpy", line 529, in <module>
    python hide:
  File "renpy/common/00keymap.rpy", line 540, in _execute_python_hide
    renpy.load(renpy.session["_reload_slot"])
Exception: Couldn't find a place to stop rolling back. Perhaps the script changed in an incompatible way?

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\bigma\Documents\RenPy\renpy\bootstrap.py", line 275, in bootstrap
    renpy.main.main()
  File "C:\Users\bigma\Documents\RenPy\renpy\main.py", line 670, in main
    run(restart)
  File "C:\Users\bigma\Documents\RenPy\renpy\main.py", line 144, in run
    renpy.execution.run_context(True)
  File "C:\Users\bigma\Documents\RenPy\renpy\execution.py", line 955, in run_context
    context.run()
  File "renpy/common/00start.rpy", line 212, in script call
    call _load_reload_game from _call__load_reload_game_1
  File "renpy/common/00keymap.rpy", line 529, in script
    python hide:
  File "/home/tom/ab/renpy-build-fix/tmp/install.linux-x86_64/lib/python3.9/site-packages/future/utils/__init__.py", line 444, in raise_
  File "renpy/common/00start.rpy", line 212, in script call
    call _load_reload_game from _call__load_reload_game_1
  File "renpy/common/00keymap.rpy", line 529, in script
    python hide:
  File "C:\Users\bigma\Documents\RenPy\renpy\ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\bigma\Documents\RenPy\renpy\python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/00keymap.rpy", line 529, in <module>
    python hide:
  File "renpy/common/00keymap.rpy", line 540, in _execute_python_hide
    renpy.load(renpy.session["_reload_slot"])
  File "C:\Users\bigma\Documents\RenPy\renpy\loadsave.py", line 816, in load
    log.unfreeze(roots, label="_after_load")
  File "C:\Users\bigma\Documents\RenPy\renpy\rollback.py", line 1105, in unfreeze
    self.rollback(0, force=True, label=label, greedy=greedy, on_load=True)
  File "C:\Users\bigma\Documents\RenPy\renpy\rollback.py", line 905, in rollback
    self.load_failed()
  File "C:\Users\bigma\Documents\RenPy\renpy\rollback.py", line 823, in load_failed
    raise Exception("Couldn't find a place to stop rolling back. Perhaps the script changed in an incompatible way?")
Exception: Couldn't find a place to stop rolling back. Perhaps the script changed in an incompatible way?

Windows-10-10.0.19045 AMD64
Ren'Py 8.1.3.23091805
Mind the School 0.1.2
Fri Dec  1 13:39:49 2023
